packages/modular-scripts/src/__tests__/esmView.test.ts

```diff
@@ -195,8 +195,7 @@ describe('modular-scripts', () => {
         stdio: 'inherit',
         env: {
           USE_MODULAR_ESBUILD: 'true',
-          EXTERNAL_CDN_TEMPLATE:
-            'https://mycustomcdn.net/[name]?version=[version]',
+          EXTERNAL_CDN_TEMPLATE: 'https://mycustomcdn.net/[name]@[version]',
```

Comment:
nit: you've used this string about 5 times, maybe just use a variable for `EXTERNAL_CDN_TEMPLATE`

Reply:
We said that we wanted to leave it like this because it's more immediate to read and the tests (we have to be DRY but tests need to be legible more)